docs/source/user-guide/tuning.md

```diff
@@ -143,28 +182,20 @@ Once it is disabled, Comet will fall back to the default Spark shuffle manager.
 
 ### Shuffle Mode
 
-Comet provides three shuffle modes: Columnar Shuffle, Native Shuffle and Auto Mode.
```

Comment:
The shuffle mode configs are internal configs intended for testing so I have removed all references to them. User's should never need to choose shuffle modes.